libmscore/rest.cpp

```diff
@@ -617,10 +624,11 @@ qreal Rest::downPos() const
 
 void Rest::scanElements(void* data, void (*func)(void*, Element*), bool all)
       {
-      func(data, this);
       ChordRest::scanElements(data, func, all);
       for (Element* e : _el)
             e->scanElements(data, func, all);
+      if (!this->trulyInvisible())
```

Comment:
using "this" in this context looks strange to a c++ programmer. I would simple omit it.